### PR TITLE
Toyota: Support Avalon 2022 (TSS2.5)

### PR DESCRIPTION
**Pull request by sshane**
Working with `ncclark#7119` on Discord to support his 2022 Avalon. Awaiting a test route

### CODE REVIEW

File: selfdrive/car/toyota/interface.py [hunk 1]
```diff
@@ -270,7 +270,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
     if ret.enableGasInterceptor:
       set_long_tune(ret.longitudinalTuning, LongTunes.PEDAL)
     elif candidate in [CAR.COROLLA_TSS2, CAR.COROLLAH_TSS2, CAR.RAV4_TSS2, CAR.RAV4H_TSS2, CAR.LEXUS_NX_TSS2,
-                       CAR.HIGHLANDER_TSS2, CAR.HIGHLANDERH_TSS2, CAR.PRIUS_TSS2]:
+                       CAR.HIGHLANDER_TSS2, CAR.HIGHLANDERH_TSS2, CAR.PRIUS_TSS2, CAR.AVALON_TSS2]:
```
Comment (sshane) [inline selfdrive/car/toyota/interface.py:273]: Is adding all new TSS2 cars to this tune the plan or should it be a separate PR? I made no verifications or gathered any data.

Comment (adeebshihadeh) [reply selfdrive/car/toyota/interface.py:273]: think this should be the default now, right @HaraldSchafer?

Comment (HaraldSchafer) [reply selfdrive/car/toyota/interface.py:273]: It isn't yet, I have not merged that PR yet.

Comment (HaraldSchafer) [reply selfdrive/car/toyota/interface.py:273]: merged

**Comment by sshane** [comment]
Added a test route!
